aten/src/ATen/core/DistributionsHelper.h

```diff
@@ -186,21 +186,26 @@ struct normal_distribution {
     T stdv;
 };
 
+template <typename T>
+struct BernoulliType { using type = float; };
```

Comment:
Let's combine `GeometricType` and `BernoulliType` to one `DiscreteDistributionType` and use it in `bernoulli_kernel` and `geometric_kernel`

Reply:
addressed!